ui/src/me/components/account/TokenRow.tsx

```diff
@@ -53,6 +63,24 @@ class TokenRow extends PureComponent<Props> {
     )
   }
 
+  private get isTokenEnbled(): boolean {
+    const {auth} = this.props
+    if (auth.status === Authorization.StatusEnum.Active) {
```

Comment:
this could be just `return auth.status === Authorization.StatusEnum.Action`

Reply:
👍